src/common/xo/index.js

```diff
@@ -289,6 +289,12 @@ export const removeServer = ({ id }) => (
   )
 )
 
+// Pool --------------------------------------------------------------
+
+export const editPool = ({ id }, props) => (
+  _call('pool.set', { ...props, id })
```

Comment:
Feel free to do so if you want but my goal is to have at least the new code written correctly.